privacy_evaluator/classifiers/classifier.py

```diff
@@ -32,34 +33,43 @@ def predict(self, x: np.ndarray):
         """
         return self.art_classifier.predict(x)
 
+    def to_art_classifier(self):
+        """Converts the classifier to an ART classifier.
+
+        :return: Converted ART classifier.
+        """
+        return self.art_classifier
+
     @staticmethod
     def _to_art_classifier(
         classifier: Union[tf.keras.Model, torch.nn.Module],
+        loss: Union[tf.keras.losses.Loss, torch.nn.modules.loss._Loss],
         nb_classes: int,
         input_shape: Tuple[int, ...],
     ) -> Union[TensorFlowV2Classifier, PyTorchClassifier]:
-        """Converts a classifier to an ART classifier.
+        """Initializes an ART classifier.
 
-        :param classifier: Classifier to be converted. Either a Pytorch or Tensorflow classifier.
+        :param classifier: Original classifier, either Pytorch or Tensorflow.
```

Comment:
This description is not correct. You are not as general to accept TensorFlow classifiers. (and please write TensorFlow instead of Tensorflow). You check if it is a keras model. But I would rather like to see the code changed instead of the documentation: (see next comment)

Reply:
We now changed it to a more general acceptance of TensorFlow models (and also losses).